src/UserSettingsPage.cpp

```diff
@@ -229,7 +231,44 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
         themeOptionLayout_->addWidget(themeLabel_);
         themeOptionLayout_->addWidget(themeCombo_, 0, Qt::AlignBottom | Qt::AlignRight);
 
+        auto encryptionLayout_ = new QVBoxLayout;
+        encryptionLayout_->setContentsMargins(0, OptionMargin, 0, OptionMargin);
+
+        QFont monospaceFont = QFont(font);
+        monospaceFont.setFamily("Courier New");
+        monospaceFont.setStyleHint(QFont::Courier);
+
+        auto deviceIdWidget = new QHBoxLayout;
+        deviceIdWidget->setContentsMargins(0, OptionMargin, 0, OptionMargin);
+
+        auto deviceIdLabel = new QLabel(tr("Device ID"), this);
+        deviceIdLabel->setFont(font);
+        deviceIdValue_ = new QLabel(QString("Loading..."));
+        deviceIdValue_->setTextInteractionFlags(Qt::TextSelectableByMouse);
+        deviceIdValue_->setFont(monospaceFont);
+        deviceIdWidget->addWidget(deviceIdLabel, 1);
+        deviceIdWidget->addWidget(deviceIdValue_);
+
+        auto deviceFingerprintWidget = new QHBoxLayout;
+        deviceFingerprintWidget->setContentsMargins(0, OptionMargin, 0, OptionMargin);
+
+        auto deviceFingerprintLabel = new QLabel(tr("Device Fingerprint"), this);
+        deviceFingerprintLabel->setFont(font);
+        deviceFingerprintValue_ = new QLabel(QString("Loading..."));
+        deviceFingerprintValue_->setTextInteractionFlags(Qt::TextSelectableByMouse);
+        deviceFingerprintValue_->setFont(monospaceFont);
+        deviceFingerprintWidget->addWidget(deviceFingerprintLabel, 1);
+        deviceFingerprintWidget->addWidget(deviceFingerprintValue_);
+
+        encryptionLayout_->addLayout(deviceIdWidget);
+        encryptionLayout_->addLayout(deviceFingerprintWidget);
+
         font.setWeight(65);
+
+        auto encryptionLabel_ = new QLabel(tr("Encryption"), this);
```

Comment:
Could you add some top padding on this section so it will appear a little more distinct, like `General`? Also it needs to be capitalized.